frontend/src/routes/healthz/+server.ts

```diff
@@ -0,0 +1,18 @@
+import type { RequestHandler } from '@sveltejs/kit';
+import axios from 'axios';
+
+export const GET: RequestHandler = async () => {
+	const backendOK = await axios
+		.get(process!.env!.INTERNAL_BACKEND_URL + '/healthz')
+		.then(() => true, () => false);
+
+	return new Response(
+		backendOK ? `{"status":"HEALTHY"}` : `{"status":"UNHEALTHY"}`,
```

Comment:
```suggestion
		JSON.stringify({ status: backendOK ? 'HEALTHY' : 'UNHEALTHY' }),
```

Wouldn't this be a bit cleaner?

Reply:
Sorry I just saw this. Since these are constants, essentially, it's probably faster to skip the overhead of re-encoding as JSON on every call. A bit of a micro-optimization.